server/controllers/stock/reports/stock/lot_barcode/lot_barcode.handlebars

```diff
@@ -0,0 +1,13 @@
+{{> head title="LOTS.BARCODE_FOR_LOT" }}
+
+<!-- body  -->
+<div class="text-center">
+  {{#if metadata.enterprise.settings.enable_barcodes}}
+    <small>{{ details.inventory_text }}</small><br>
```

Comment:
I don't think the text needs to be "small" here.  I suggest removing the 'small' tags.  There is plenty of room on the document!